.github/workflows/test.yaml

```diff
@@ -0,0 +1,41 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
```

Comment:
This is great. In general, if you could make it have the same format as https://github.com/googleapis/java-spanner/blob/master/.github/workflows/ci.yaml (and give it the same name), that would be awesome.

Reply:
OK!